### PR TITLE
Curated Hiroko's instructor page

### PR DESCRIPTION
**Pull request by lsminter**
I'm really happy with how this turned out! It felt good to be able to do this pretty much on my own. Using sanity really made this easy. 

Screen Shot 2021-06-21 at 12 40 25 PM

Hiroko has two courses on egghead and wrote her own eBook that I added to her instructor page.

### CODE REVIEW

**Comment by dealingwith** [comment]
This probably isn't specific to this page, but in light mode this card effect makes the text hard to read

Screen Shot 2021-06-21 at 3 10 48 PM

**Comment by dealingwith** [comment]
@vojtaholik see above

Otherwise this LGTM

File: src/components/search/instructors/hiroko-nishimura/index.tsx [hunk 1]
```diff
@@ -1,20 +1,146 @@
...
+      >
+        <div className="relative z-10 px-5 sm:py-16 py-10 sm:text-left text-center">
+          <div className="space-y-5 mx-auto flex items-center justify-center max-w-screen-xl">
+            <div className="flex flex-col items-center justify-center sm:space-x-5 sm:space-y-0 space-y-5 gap-10 mt-10">
+              <div className="flex-shrink-0">
+                <Link href={path}>
+                  <a
+                    tabIndex={-1}
+                    onClick={() =>
+                      track('clicked jumbotron resource', {
```
Comment (zacjones93) [inline src/components/search/instructors/hiroko-nishimura/index.tsx:105]: this track link is inconsistent with the `eventName` above on line 77
```suggestion
                      track('clicked Hiroko Nishimura instructor page CTA', {
```

File: src/components/search/instructors/hiroko-nishimura/index.tsx [hunk 2]
```diff
@@ -1,20 +1,146 @@
...
+              </div>
+              <div className="flex flex-col sm:items-start items-center">
+                <h2 className="text-xs text-white text-opacity-80 uppercase font-semibold mb-2">
+                  {byline}
+                </h2>
+                <Link href={path}>
+                  <a
+                    className="text-xl font-extrabold leading-tighter text-white hover:text-blue-300"
+                    onClick={() =>
+                      track('clicked jumbotron resource', {
```
Comment (zacjones93) [inline src/components/search/instructors/hiroko-nishimura/index.tsx:129]:
```suggestion
                      track('clicked Hiroko Nishimura instructor page CTA', {
```

File: src/components/search/instructors/hiroko-nishimura/index.tsx [hunk 3]
```diff
@@ -1,20 +1,146 @@
...
+      'path': url,
+      path,
+      description,
+      image,
+      byline
+    }
+  }
+}`
 
+const FeaturedCourse: React.FC<{location: string; resource: any}> = ({
```
Comment (zacjones93) [inline src/components/search/instructors/hiroko-nishimura/index.tsx:70]: @dealingwith this page uses a one-off component so those darkmode style issues should be isolated to this page.

@lsminter here's an example of how to fix this
https://github.com/eggheadio/egghead-next/blob/main/src/components/search/instructors/stephanie-eckles/index.tsx#L186-L206

**Review by zacjones93 — CHANGES_REQUESTED** [review]

**Comment by lsminter** [comment]
@zacjones93 I believe I've done all of your suggested updates. @dealingwith Can you guys double check to make sure this works?

**Comment by dealingwith** [comment]
Card still has issues in light mode.

Screen Shot 2021-06-23 at 8 10 43 PM

**Comment by zacjones93** [comment]
Went ahead and got that fixed @lsminter @dealingwith 

Image 2021-06-28 at 5 02 37 PM
Image 2021-06-28 at 5 02 46 PM

**Review by zacjones93 — APPROVED** [review]

**Pull request merged**